### ISSUE TITLE
Development plan for voyage-4 

### ISSUE THREAD

**Issue by mar-bi** [chingu-voyage4/grad.then#220]
I'd like to store a development plan in a separate issue. :page_with_curl: 
We can collect, add, and discuss our ideas here until we will finally end up with MVP form.  :framed_picture:

Here is my insight on what we are able to accomplish. :muscle: 

![screenshot from 2018-02-22 09-32-20](https://user-images.githubusercontent.com/25195616/36525863-7a4ddd22-17b4-11e8-8601-129ceb0cf3f0.png)

**PS.**
Also, I'd like to add a small change to our workflow - I want everyone to pick up a most attractive for him/her feature from the MVP and work on it.
This will help to ensure an unblocking behavior of our work. So that, everyone can continue of his/her own speed without waiting for a teammate to complete a piece of the app.
Of course, everyone support each other and help as needed.

For example, I'd like to do a migration from a state in containers to the Redux architecture. We agreed about this and I'm going re-factor the code for this migration, make **PR**, then you will review it.
If I'm stack at some point, I'm going to ask for help.

I expect that this voyage will be a challenge for us. We are going to work with more advanced instruments and we are in job search. (yes, me too)
I highly appreciate our clear and friendly communications and will be very happy to congratulate you with a new job. :smile: :star2: :fireworks:

**Comment by mar-bi** [comment]
As we agreed , our first step includes:
- moving the app state to **Redux**,
- discovering the capabilities of the free tier of **Contentful**,
- setting up the environment for UI testing (**Jest**, **Enzyme**)
